### ISSUE TITLE
Logo proposal for Wild Web Developer

### ISSUE THREAD

**Issue by jabby** [eclipse/wildwebdeveloper#94]
Hello all, I think it will be a good idea to have a nice logo representing Wild Web Developer. 
It will be useful for "marketing" on the Eclipse Foundation website and on the marketplace. 
It will also help user to identify the plugin in general. 

Few weeks ago I asked on twitter if anybody want to contribute a logo. 

@RenaudGoujatDesign answered positively and want to contribute on this project for a logo. 
We already had a great meeting around how to contribute to Wild Web Developer and other Eclipse projects in general. 
We also talked about the project itself and what is my vision of the product. 

Before starting any work, I would like to know if there is any objections / advices about the contribution. 

Feel free to answer to this issue. 

I let @RenaudGoujatDesign presenting himself

**Comment by RenaudGoujat** [comment]
Hi everyone !
I am Ui and Ux designer, living in Nantes, France.
I am passionate with interface design, user experience and illustration.

I would be very happy to place my skills at your service.
I am very interested by the Open Source world.
I like the idea of letting the possibility for everyone to get involved, share knowledges, and to take the lead as part of Open Source projects, that aim to be usable and accessible for everyone.

Joining your project, and the Eclipse foundation, would be a wonderful opportunity to me to be commited to the Open source collaborative world.

I can create a reliable logo for your project, an easy to use graphic styleguide,
a landing page and motion graphics.

**Comment by mickaelistria** [comment]
That would be really welcome! Feel free to ask any question if you need for context to find out a good logo. Let me put some anyway:
* The target audience are developers (backend and frontend) who need to developer HTML/CSS/JavaScript/TypeScript in the Eclipse IDE.
* The main entry point for installation, and the main place where the logo will be seen in the Marketplace dialog inside the Eclipse IDE. There is the page with the default logo: https://marketplace.eclipse.org/content/wild-web-developer-web-development-eclipse-ide-experimental . It's usually showing up when people need this editor, and other "competitive" proposals are made, the main one we'd like to surpass (ie for which we want our logo to be more appealing: is WebTools https://marketplace.eclipse.org/content/eclipse-web-developer-tools-0 .
* The logo may include the project name or not.
* The logo can use the palette or even include derivative work from the current default logo (blue and orange orb)

**Comment by edertone** [comment]
Here are two ideas
![test2](https://user-images.githubusercontent.com/16387357/50705356-e107da00-105a-11e9-8fde-242a32cacbd6.png)
![test3](https://user-images.githubusercontent.com/16387357/50705358-e107da00-105a-11e9-867a-5124be8dce1b.png)

**Comment by mickaelistria** [comment]
Thanks. I like those. However, I feel like they're not connected enough to
the Eclipse IDE logo.
Would it be possible to reuse some graphical «signatures» of Eclipse IDE
logo like its color palette?


-- 
Mickael Istria
Eclipse IDE <https://www.eclipse.org/downloads/eclipse-packages/>
developer, for Red Hat Developers <https://developers.redhat.com/>

**Comment by edertone** [comment]
![test10](https://user-images.githubusercontent.com/16387357/50724981-d69b1e00-10f6-11e9-9973-d4be89415925.png)

**Comment by mickaelistria** [comment]
@jabby @angelozerr @dschaefer , folks, as declared stakeholders of Wild Web Developer, please comment whether it's good to you or not. If you are fine with it as a logo, give it a +1, otherwise, please share your concerns and we'll see if we can make it better for everyone.

**Comment by mickaelistria** [comment]
@PhaserEditor2D since you're closely following the project, you're also invited to vote and comment here (see previous comment).

**Comment by PhaserEditor2D** [comment]
The logo above from @edertone looks too horizontal to me, maybe something more circular would be better, it is closer to the Eclipse logo and the Wild Web (World) concept.


By the way, this link can help: https://www.eclipse.org/artwork/

**Comment by PhaserEditor2D** [comment]
Some ideas here:

![wildwebdeveloprlogospng](https://user-images.githubusercontent.com/12187299/50726192-86609380-10d7-11e9-8fe9-c7be56b2ff15.png)

Svg: [logo-wwd.zip](https://github.com/eclipse/wildwebdeveloper/files/2729625/logo-wwd.zip)

**Comment by PhaserEditor2D** [comment]
I like this one too:

![image](https://user-images.githubusercontent.com/12187299/50726297-076c5a80-10d9-11e9-9322-dd40823e15cb.png)

**Comment by PhaserEditor2D** [comment]
And more: 
![image](https://user-images.githubusercontent.com/12187299/50726459-2f5cbd80-10db-11e9-95d0-0ab6e05c26f4.png)

**Comment by mickaelistria** [comment]
@PhaserEditor2D IMO, your logo proposals are too generic and fail at embodying this particular project. The same logo could probably be used for any plugin in Eclipse IDE, we need something more specific..

**Comment by PhaserEditor2D** [comment]
@mickaelistria I see it fails on represent WWD.

Anyway, these are the WWD elements I see in the logos: 

![image](https://user-images.githubusercontent.com/12187299/50730655-6608f700-1120-11e9-803e-f758ca1a829a.png)

**Comment by RenaudGoujat** [comment]
Hi Guys,
I am Renaud, I joined your team by the month of december. I am a graphic designer, and I am still interested in creating your logo. I have been a bit busy these days, but now I am available to work for you. 

I think that you are doing it backwards, with your brand identity. It is risky to propose a logo design, and then start think about a concept with the team.

I think that the best way to create your logo is to send first a questionnaire to all the members of the project. That way, with all the informations that we will gather, it will be possible for us to define your project accurately. This will permit us to have all the elements that we need to create the logo.

I suggest you to fill my questionnaire, and send it to me the lastest for the 17th of january.
Waiting forward to hearing from you soon,

[WWD LOGO.pdf](https://github.com/eclipse/wildwebdeveloper/files/2743018/WWD.LOGO.pdf)

Cheers,
Renaud

**Comment by mickaelistria** [comment]
I've converted the form to a Google Doc: https://docs.google.com/document/d/1Mlc7IMPBAGImyj4vX1drSeFaiswW6yL1TWX7jV3qqNM/edit# . Committers should have received an invite to comment there and suggest edits.
Let's give everyone a long week before I share it here.

**Comment by RenaudGoujat** [comment]
Hi everyone,
Thank you Mickael for giving me access to yours answers.

Jabby told  me that your project is to create an integrated development environment for developers, made by real developers, who know perfectly the needs and constraints of a professional in he's day to day work. I think that you can find a lot of valuable informations and datas from your own developer experience.

You can also define who will be interested by your product, using good hypothesis. For example, is it important to catch up youngters attention, or to seduce experienced developers ? Do you think that your target is worldwide, or based in a more specific region, like western Europe ?

**Comment by mickaelistria** [comment]
> For example, is it important to catch up youngters attention, rather than seducing experienced developers ? Do you think that your target is worldwide, or based in a more specific region, like western Europe ?

I really don't want the logo to target a subpart of the target audience in particular, and while there are some stats that would tend favor a sub-population, I think those are really anomaly of the domain in term of diversity and don't want the project strategy to build on them as it's just a way to consolidate or grow this anomaly.

**Comment by PhaserEditor2D** [comment]
By the way, is there a twitter account for the project? I would love to @ it in our posts.

**Comment by mickaelistria** [comment]
No, there is not, and unlesswe have a strategy and someone has dedicated time to take care of it,I'm reluctant to create one.
It would be one more channel to monitor while we achieve more productivity by having GitHub as almost single channel.
Also,the project is «just» an Eclipse plugin, it doesn't really qualify as a product.
More effort should be placed in actual payload and ensuring Wild Web Developer can be a replacement of WTP so the user base would grow naturally.
However, we may start an account for the «Eclipse IDE for Web and JavaScript developers» when we consider it reaches a good enough quality and it relies on Wild Web Developers. Let's wait a bit and discuss it again within a month or 2 hopefully.

**Comment by edertone** [comment]
more ideas

![test12](https://user-images.githubusercontent.com/16387357/51071796-275ec980-1657-11e9-8489-bddb615ae755.png)
![test11](https://user-images.githubusercontent.com/16387357/51071797-275ec980-1657-11e9-85b3-cc81e7b59945.png)

**Comment by wimjongman** [comment]
I love it.  I love its mean look. I also love your text. However, horse heads don't make webs about honoring the real makers of the web:

# SPIDERS!!!

16
![wildewebspider16](https://user-images.githubusercontent.com/180969/51439790-b1460c80-1cbf-11e9-8952-5c1c14e576c6.png)

16@2
![wildewebspider32](https://user-images.githubusercontent.com/180969/51439796-c6bb3680-1cbf-11e9-972e-a0cf42440bc7.png)

16@3
![wildewebspider48](https://user-images.githubusercontent.com/180969/51439802-d63a7f80-1cbf-11e9-9e85-d94e963c69ef.png)

Logo
![wildewebspider5](https://user-images.githubusercontent.com/180969/51439809-df2b5100-1cbf-11e9-900d-b720374f2689.png)

With text
![wildewebspider4](https://user-images.githubusercontent.com/180969/51439814-eb171300-1cbf-11e9-96a3-ea02c0a39561.png)

![wildewebspider3](https://user-images.githubusercontent.com/180969/51439817-f9fdc580-1cbf-11e9-986f-cfce2e219d12.png)

![wildewebspider2](https://user-images.githubusercontent.com/180969/51439820-fff3a680-1cbf-11e9-8ae9-5dce9ef3389e.png)

![wildewebspider](https://user-images.githubusercontent.com/180969/51439827-06821e00-1cc0-11e9-8415-2d44284fd38e.png)

## SVG
[wildewebspiderdev.zip](https://github.com/eclipse/wildwebdeveloper/files/2776643/wildewebspiderdev.zip)

**Comment by wimjongman** [comment]
## Wilde West

![cowboy](https://user-images.githubusercontent.com/180969/51440063-eef86480-1cc2-11e9-98ed-5f3f4721edf6.png)


![cowboy2](https://user-images.githubusercontent.com/180969/51440066-f3bd1880-1cc2-11e9-888e-c39622c9dc40.png)

## SVG
[wildewebdev.zip](https://github.com/eclipse/wildwebdeveloper/files/2776663/wildewebdev.zip)

**Comment by edertone** [comment]
![test17](https://user-images.githubusercontent.com/16387357/51492036-ad45e780-1db0-11e9-8743-c7c909c38145.png)

**Comment by mickaelistria** [comment]
Does anyone want to submit more logos, or can we start a vote?

**Comment by RenaudGoujat** [comment]
Hi everyone. I am still interested to work with you. But I since I have sent a questionnaire, in order to receive informations from all the members of the project to create a reliable logo, only Mickael sent us back his answers. I still think that multiply random logo proposals without reaching everyone's agreement on the right direction to choose (the right keywords, the right visual concept) won't permit us to achieve a good logo.

**Comment by edertone** [comment]
Just another variation of a previous logo

![test21](https://user-images.githubusercontent.com/16387357/51769297-067c8680-20e3-11e9-875f-b34f93717d5c.png)

**Comment by mickaelistria** [comment]
> Hi everyone. I am still interested to work with you. But I since I have
> sent a questionnaire, in order to receive informations from all the members
> of the project to create a reliable logo, only Mickael sent us back his
> answers.
>


Hi Renaud,
I think you can assume that no additional answer is a form either of
agreement or of lack in interest, and after this time just proceed with the
current answers as they're all you'll get.
To be clear, this logo is mostly a piece of detail as the real product is
and will be the Eclipse IDE that already has logo and branding. That's one
reason why people may not be so involved in providing more input or in
qualifying the mission and constraints of the logo.



-- 
Mickael Istria
Eclipse IDE <https://www.eclipse.org/downloads/eclipse-packages/>
developer, for Red Hat Developers <https://developers.redhat.com/>

**Comment by RenaudGoujat** [comment]
Hi Everyone,
Hi Mikael,

Ok I will use you and Gautier advices and informations as basis for my logo proposal.
I won't be available to work on the logo this week, would it be ok if I send you a proposal for the 11th of january ?

Let me know if this deadline is good for you.

Have a nice day,
Renaud

**Comment by jabby** [comment]
Hi Renaud, 

There is no problem if you can't work this week on your proposal. I'm ok with the 11th of february. 

Thanks for your work.

**Comment by PhaserEditor2D** [comment]
I agree with the Mickael document. I just added two lines.

**Comment by RenaudGoujat** [comment]
Hi everyone, I need some time to fine tune some details, I will send my proposals tomorrow.

**Comment by RenaudGoujat** [comment]
Hi Everyone,
Please find attached my three proposals. I have tried to keep in mind these main ideas : "Global", "Efficiency", "Code faster, Code better", "creativity", "modern", "empower".

**Comment by RenaudGoujat** [comment]
![plan de travail 1](https://user-images.githubusercontent.com/45291042/52676652-26a5a580-2f2b-11e9-8009-f8f91551c32e.png)

**Comment by RenaudGoujat** [comment]
![plan de travail 3](https://user-images.githubusercontent.com/45291042/52676663-302f0d80-2f2b-11e9-9284-3a8ad97b27e2.png)

**Comment by RenaudGoujat** [comment]
![plan de travail 3](https://user-images.githubusercontent.com/45291042/52676773-8a2fd300-2f2b-11e9-9734-87df0e821e84.png)

**Comment by RenaudGoujat** [comment]
![plan de travail 1](https://user-images.githubusercontent.com/45291042/52676874-eeeb2d80-2f2b-11e9-94e3-b3667bcaa983.png)

**Comment by RenaudGoujat** [comment]
![plan de travail 3](https://user-images.githubusercontent.com/45291042/52676887-f8749580-2f2b-11e9-81fb-9dcfb5e895fe.png)

**Comment by RenaudGoujat** [comment]
![plan de travail 4](https://user-images.githubusercontent.com/45291042/52676895-05918480-2f2c-11e9-87b0-66bc462b4ae1.png)

**Comment by dschaefer** [comment]
I'm a big fan of this one, Piste A. Very professional and modern.

**Comment by jabby** [comment]
Same as @dschaefer I'm falling in  love with "Piste A".

**Comment by PyvesB** [comment]
Piste A is indeed very nice, however, it reminds me of the Western Digital brand logo.

**Comment by mickaelistria** [comment]
I like A as well. It's my favorite logo of all proposals.
I don't mind ressemblance with Western Digital. If this is the logo we favor, we can ask the Eclipse IP team to check whether it's a legal issue.

**Comment by wimjongman** [comment]
Clearly, @RenaudGoujat is a very skilled designer and agreed, "A" is visually appealing. However, there is nothing "Wild" to be seen here.  It is very mainstream and corporate IMHO. I was hoping for a more exciting logo that does right to the project name. In addition, how will this translate to square icons?

I think there is more promise in "B" (although the christmas must be the most domesticated ;) 
However, the three mini icons in this logo are a cool idea. They can serve a purpose outside of the main logo. Which unfortunately again is an acronym (double-u-double-u-dee does not sound very satisfying).

I also like to see some more work on logo "C". It has the most potential IMHO but it is hard to recognize the W. Also the current shape is too smooth. It should be much rougher. It is the most useful as a square icon as well.

These are just my opinions. Please dismiss them if you like.

**Comment by mickaelistria** [comment]
> These are just my opinions. Please dismiss them if you like.

Any feedback is welcome at this point. Next week, we'll start discussing a selection process if there seems to be need for it (but currently, it seems like A still is the favorite of majority of committers+mentor)

**Comment by edertone** [comment]
I would like to say that for me, the most original and the one that most resembles to the meaning of this plugin is the one from [wimjongman] that shows a cowboy riding a horse. However, i think that it should be refined and polished to make it more stylistic and modern. 

Maybe changing the typo and refining the cowboy drawing could make it the winner one...

But that logo is for me the best.

**Comment by mickaelistria** [comment]
Hi all,

It seems like project mentor and 2 project committers out of 3 agree on using logo submitted by @RenaudGoujat as "Piste A".
I'll start the process of adoption by asking IP team whether they think we can adopt it or about whether they see some issues with it.

**Comment by mickaelistria** [comment]
Logo has been approved for usage by the IP team.
We can proceed with adoption: adding it to README, to marketplace, and so on.

**Comment by mickaelistria** [comment]
Hi @RenaudGoujat , thanks a lot for your work.
Would you mind contributing the various files related to this logo (including some vectorial ones ideally) inside the project tree (in a `logo` folder) and submit them as a pull request ?

**Comment by RenaudGoujat** [comment]
Hi @mickaelistria, thank you for your nice feedback. Would it be possible to have a precise list of the graphic elements that you will need ? I plan send you the logo in different sizes and formats with a favicon, but will I need to add other elements ? Thank you everyone for giving me the opportunity to contribute to your project.

**Comment by mickaelistria** [comment]
I think we'll need the "banner" (logo + name) in the size you already provided, and also in vectorial project (.ai or .xcf) in case we need to extract derive it in the future for use-case we didn't think about yet.
Then, it's mostly square icons (with transparent background) for light background in different sizes (16, 32, 64, 128, 256), and also square icon with background for cases where the background color is variable and not reliable, in same sizes.
HTH

**Comment by mickaelistria** [comment]
@RenaudGoujat Feel free to tell me if you need more details than what's in the previous comment. Im the meantime, I'd just extract the logo and banner from your proposal to put it in README and in Marketplace. We'll replace it with better versions ASAP.

**Comment by RenaudGoujat** [comment]
[WWD_LOGO.zip](https://github.com/eclipse/wildwebdeveloper/files/2962798/WWD_LOGO.zip)

Hi Everyone,

I didn't manage to create a new pull request.

Please find attached the WWD logo in different sizes, image file formats and colors. 
Let me know if it is ok.

Have a nice day,
Renaud

**Comment by mickaelistria** [comment]
Thank you very much @RenaudGoujat ! As those are big files, I've not put them in the repo, but instead put a README with a link (see https://github.com/eclipse/wildwebdeveloper/commit/6915c75581dcdf38230c60424124cdeac386c557 )
I did update the Marketplace entry.

**Comment by PyvesB** [comment]
Looks great on the marketplace! 👍
